src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -273,6 +276,14 @@ class GlobalSelectionHeader extends React.Component {
       }
     }
 
+    // Projects have finished loading so now we can enforce a single project
+    if (prevProps.loadingProjects && !this.props.loadingProjects) {
+      const {project} = getStateFromQuery(location.query);
+      if (!project) {
+        this.enforceSingleProject({forceProject});
```

Comment:
@billyvg I think this could potentially be called again in a specific circumstance when an org does no have multi-project selection feature. I've worked around this in https://github.com/getsentry/sentry/pull/15977/commits/efd69c6c3633649ce5be40a45edf01c4acbfa8fd 

What do you think?

Reply:
Makes sense, thanks!

Reply:
Hmm tested this on staging and it's doing unwanted things

Reply:
@billyvg oh yikes 😬  Feel free to revert my changes.